src/lib.rs

```diff
@@ -151,6 +151,8 @@ pub type UuidBytes = [u8; 16];
 /// The version of the UUID, denoting the generating algorithm.
 #[derive(Debug, PartialEq, Copy, Clone)]
 pub enum UuidVersion {
+    /// Special case for `nil` [`struct.Uuid.html`].
```

Comment:
What do we have the `[`struct.Uuid.html`] here for? Does this render in the docs as `struct.Uuid.html`?

Reply:
Woops I forgot to link it will fix it ASAP

Reply:
That was supposed to be [`Uuid`](struct.Uuid.html)

Reply:
@KodrAus fixed.. can you approve